content/ja/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -28,7 +28,7 @@ Deploymentによって作成されたReplicaSetを管理しないでください
 
 下記の項目はDeploymentの典型的なユースケースです。
 
-* ReplicaSetをロールアウトするために[Deploymentの作成](#creating-a-deployment)を行う: ReplicaSetはバックグラウンドでPodを作成します。Podの作成が完了したかどうかは、ロールアウトのステータスを確認してください。  
```

Comment:
I dont think these changes are necessary. The blanks are set on purpose as it means the same as <br> tag in HTML.

Reply:
Ahh, my editor removed that whitespaces... 
I pushed commit to restore this.